### PR TITLE
304 page title component

### PR DESCRIPTION
**Pull request by zlrkw11**
closes

### CODE REVIEW

**Comment by zlrkw11** [comment]
image
- figma styling for title should be h2 not h1

File: client/src/components/generic/PageTitle/PageTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export interface IPageTitleProps {
+  title: string
+}
+type props = IPageTitleProps
+
+export const PageTitle = ({ title }: props) => {
+  return (
+    <div className="bg-light-blue-100 flex h-[70px] w-full items-center justify-center">
+      <div className="text-gray-1 italic">
```
Comment (bcho892) [inline client/src/components/generic/PageTitle/PageTitle.tsx:9]: can we apply this directly to the `h2`

File: client/src/components/generic/PageTitle/PageTitle.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export interface IPageTitleProps {
+  title: string
+}
+type props = IPageTitleProps
+
+export const PageTitle = ({ title }: props) => {
+  return (
+    <div className="bg-light-blue-100 flex h-[70px] w-full items-center justify-center">
+      <div>
```
Comment (bcho892) [inline client/src/components/generic/PageTitle/PageTitle.tsx:9]: Is the div still needed with that change?

Comment (zlrkw11) [reply client/src/components/generic/PageTitle/PageTitle.tsx:9]: I have a habit of wrapping everything into a div. Removed now.

**Review by bcho892 — APPROVED** [review]

**Pull request merged**